app/assets/stylesheets/components/_cardtrip.scss

```diff
@@ -0,0 +1,44 @@
+.card-trip {
+  overflow: hidden;
+  background: white;
+  box-shadow: 0 0 15px rgba(0,0,0,0.2);
+  padding-left: 10px;
+}
+
+
+.card-trip h2 {
+  font-size: 16px;
+  font-weight: bold;
+  margin: 0;
+  color:#0A4B57;
+}
+
+.card-trip-infos p {
+  font-size: 14px;
+  opacity: .7;
+  margin: 0;
+}
+
+.card-info {
+  display: flex;
+  justify-content: space-between;
+  align-content: flex-end;
+  padding: 20px;
+  color: $dark-blue;
+
+  img {
+    border: solid 3px $dark-blue;
+    width: 60px;
+    height: 60px;
+  }
+
+  p {
+    font-size: 20px;
+  }
+
+}
+
+.card {
```

Comment:
Change the class name because this is a bootstrap class and could cause some problems.